src/utils/evm.ts

```diff
@@ -75,10 +76,13 @@ export function getEvmTxLink(
     return isApproveTx
       ? `https://etherscan.io/tx/${txHash}`
       : `https://wormholescan.io/#/tx/${txHash}`
-  } else if (key === "moonbeam") {
-    return `https://moonbeam.subscan.io/tx/${txHash}`
```

Comment:
have you tested that link?

Reply:
yes, but it goes to regular EVM explorer instead of Subscan https://moonscan.io/tx/0x7cd55ff3310eae17b454a9fcb34682fde185d444119d06f100a90e68a939b367